source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,570 @@
+
+We recommend setting at least one, achievable and time-based outcome.
+
+At the same time, you should begin thinking about technical and security requirements. A great place to start is with an implementation plan. So, feel free to `use our template available <https://docs.mattermost.com/getting-started/implementation_plan.html>`_ in the Mattermost documentation. Also, begin by engaging with your technical and security questions like those below.
+
+- Are there any known security or access requirements?
+- Is a VPN connection required?
+- Do we use an Enterprise Mobile Management (EMM) Provider?
+
+Again, this means pausing before you jump into your Mattermost mobile deployment. This takes a little more time, but will make the difference in ensuring your see a return on your investment.
+
+----
+
+###################################
+Mattermost Mobile Deployment Primer
+###################################
+
+A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
+
+We’re here to support you.
+
+This primer serves as an introduction to Mattermost mobile technologies. It is a complement to our technical documentation, and focuses on three main areas:
+
+- Technical Requirements
+- Basic Features and Technologies
+- Trouble Spots
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
+
+**********************
+Technical Requirements
+**********************
+
+`Supported Mattermost Server Versions <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ 
+    Minimum requirements for the Mattermost Server is maintained in our documentation (link above). 
+    **However, we do recommend running the latest version** of the Mattermost Server as it contains the most recent features and any applicable security updates.
+
+    If this is not possible, we encourage you to be on at least version 5.12 of the Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
+
+`Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`_
+    Basic mobile device requirements are provided in the link above.
+
+********************************
+Public App Store vs. Custom Apps
+********************************
+
+The most critical decision you will make in your mobile deployment will be whether to use the apps provided by Mattermost via Google Play and Apple App Store, or to build and distribute custom versions of the Mattermost apps. Below we provide a very general overview of these options.
+
+Public App Store
+================
+
+Utilizing the app distributed by Mattermost in Google play and the Apple App Store greatly reduces the deployment time and is our recommended approach. Key benefits include:
```

Comment:
Should we link to the stores web page for our app? https://apps.apple.com/us/app/mattermost/id1257222717 0/5

Reply:
Don't think this would add much value for the audiece. 0/5 if we want to point to our doc page for the mobile apps https://docs.mattermost.com/mobile/mobile-appstore-install.html.